commandline/src/main/java/tech/pegasys/web3signer/commandline/subcommands/Eth2SubCommand.java

```diff
@@ -41,13 +36,31 @@
       arity = "1")
   private boolean slashingProtectionEnabled = false;
 
+  @Option(
+      names = {"--slashing-protection-db-url"},
+      description = "The jdbc url to use to connect to the slashing protection database",
+      paramLabel = "<storage label>",
```

Comment:
storage label ??? you meant `paramLabel = "<jdbc url>"` ??

Reply:
yes I did. done